### DISCUSSION TITLE
SVG Procedural Node System with animations  web support

### DISCUSSION THREAD

**Discussion by Dinesh0N** [friction2d/friction#364] [Ideas]
SVG Procedural Node System with ( animations and Interactive ) SVG web support 

Procedural Nodes (Graph-based tools) like 
Animations in design tools (e.g., Blender and Graphite.rs).
This uses a visual interface where animations and interactions are created by connecting nodes that represent operations, properties, or logic.

Advantages:

User-Friendly: No programming required; drag-and-drop interface is more intuitive for artists/designers.

Visualization: Provides a clear overview of animation logic through connected nodes.

Reusability: Easily reuse animation setups across different projects.

Rapid Prototyping: Great for quick mockups or iterative design.

Consistency: Node-based systems enforce structure, reducing logic errors.

Nodes: basic required nodes included expression node(advance scripting).

**Comment by AlmostMikhail** [comment]
This seems like a huge task. I don't even know where you would start.
